### PR TITLE
Seatmap improvements

### PR DESCRIPTION
**Pull request by kul3r4**
- add space around the seatmap in which you can still double tap
- fix the point of zooming by adding `layout=fill` to the `amp-selector`

@sebastianbenz the `div` strategy is necessary as explained in the comment as adding a padding directly to the svg interacts with the `amp-pan-zoom` sizing, while just having a margin outside the `amp-pan-zoom` makes the margin not clickable for zooming.

@pbalaram FYI

### CODE REVIEW

**Comment by kul3r4** [comment]
@sebastianbenz ping

File: src/30_Advanced/SeatMap.html [hunk 1]
```diff
@@ -79,6 +79,9 @@
           right: 0;
           margin: auto;
       }
+      .seatmap  amp-selector .svg-container {
+        padding: 0 20px 0 20px;
```
Comment (sebastianbenz) [inline src/30_Advanced/SeatMap.html]: shorter: `padding: 0 20px`.

Comment (sebastianbenz) [reply src/30_Advanced/SeatMap.html]: note, we usually use multiples of 8 for padding (0.5 rem). I that case I'd suggest using 24px (if it fits with the sample).

Comment (kul3r4) [reply src/30_Advanced/SeatMap.html]: done

**Comment by kul3r4** [comment]
@sebastianbenz PTAL

**Pull request merged**